sentry/src/main/java/io/sentry/SentryEnvelopeItem.java

```diff
@@ -148,6 +150,49 @@ public static SentryEnvelopeItem fromUserFeedback(
     return new SentryEnvelopeItem(itemHeader, cachedItem::getBytes);
```

Comment:
H: consider changing this too
https://github.com/getsentry/sentry-java/pull/1082/files#r535112346

Reply:
Changed it here, thanks: https://github.com/getsentry/sentry-java/pull/1091